src/components/pc-overlay.vue

```diff
@@ -1,138 +1,239 @@
+            starting an experiment, then we will increase the false positive
+            rate of our experiments beyond the standard 10% at Booking.com. This
+            calculator assumes that the traffic is equally split between all
+            variants (e.g. 50%/50%). If your experiment has multiple stages,
+            then the values that you enter (e.g. base rate, sample size) should
+            be for the decision making stage.
+          </li>
+          <li>
+            For customer-facing experiments, runtime should be a based on full
+            week cycles, e.g. 7, 14, 21, etc days. See
+            <a
+              href="https://www.google.com/url?q=https://wiki.booking.com/pages/viewpage.action?pageId%3D317526288&amp;sa=D&amp;ust=1510155767728000&amp;usg=AFQjCNEBPONnocKI29b0FniJPNElqKcMZA"
+              >this wiki page</a
+            >
+            for more details.
+          </li>
+          <li>
+            The calculator also requires values for the false positive rate and
+            power, which at Booking.com are set at 10% and 80% respectively. Do
+            not &nbsp;change these values unless you have an advanced
+            understanding of experimentation.
+          </li>
+          <li>
+            Note that you cannot meaningfully use this power calculator to
+            determine the power of an experiment after it has been started. See
+            <a
+              href="https://www.google.com/url?q=http://www.tc.umn.edu/~alonso/Hoenig_AmericanStat_2001.pdf&amp;sa=D&amp;ust=1510155767729000&amp;usg=AFQjCNGSC5HLs1MayGDXdukvUiefE8-Xpw"
+              >this technical paper</a
+            >
+            or
+            <a
+              href="https://www.google.com/url?q=https://en.wikipedia.org/wiki/Statistical_power%23A_priori_vs._post_hoc_analysis&amp;sa=D&amp;ust=1510155767730000&amp;usg=AFQjCNHb-ejFukphEfKhr7QFMr-n637SQw"
+              >this wikipedia summary</a
+            >
+            for more information.
+          </li>
+        </ul>
+      </div>
+      <button v-on:click="closeOverlay" class="pc-overlay-close">
+        <svg
+          version="1.1"
+          baseProfile="full"
+          width="20"
+          height="20"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <circle cx="50%" cy="50%" r="50%" fill="#e2e6eb" />
+
+          <line
+            x1="30%"
+            y1="30%"
+            x2="70%"
+            y2="70%"
+            stroke-width="2"
+            stroke="#5d5f63"
+          />
+          <line
+            x1="70%"
+            y1="30%"
+            x2="30%"
+            y2="70%"
+            stroke-width="2"
+            stroke="#5d5f63"
+          />
+
+          <desc>Close</desc>
+        </svg>
+      </button>
     </div>
-
+  </div>
 </template>
 
 <script>
 export default {
-    template: '#pc-overlay',
-    props: ['isoverlayopen'],
-    data () {
-        return {}
+  template: '#pc-overlay',
+  props: ['isoverlayopen'],
+  data() {
```

Comment:
You can remove this empty data

Reply:
To be honest, I believe this component is dead code and can be safely removed.